apps/lockfile-explorer-web/src/containers/LockfileViewer/index.tsx

```diff
@@ -71,22 +73,30 @@ const multipleVersions = (entries: LockfileEntry[]): boolean => {
 };
 
 export const LockfileViewer = (): JSX.Element | ReactNull => {
+  const dispatch = useAppDispatch();
   const [filter, setFilter] = useState('');
   const entries = useAppSelector(selectFilteredEntries);
   const activeFilters = useAppSelector((state) => state.entry.filters);
-  const updateFilter = useCallback((e) => setFilter(e.target.value), []);
+  const updateFilter = useCallback((e) => {
+    setFilter(e.target.value);
+    saveFilterToLocalStorage(e.target.value);
+  }, []);
+  const pop = useCallback(() => {
```

Comment:
The `Back`/`Forward` buttons don't seem to be working correctly for me... let me know if you need more specific repro steps.

Reply:
Fixed!